documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -65,12 +65,12 @@ spec:
 Cruise Control uses _capacity limits_ to determine if certain resource-based optimization goals are being broken. 
 
 You specify capacity limits for Kafka broker resources in the `brokerCapacity` property in `Kafka.spec.cruiseControl` .
-Capacity limits can be set for the following broker resources in the described units:
+Capacity limits can be set for the following broker resources in the described units (the "i" in the units is optional):
 
-* `disk`            - Disk storage in bytes
-* `cpuUtilization`  - CPU utilization as a percent (0-100)
-* `inboundNetwork`  - Inbound network throughput in bytes per second
-* `outboundNetwork` - Outbound network throughput in bytes per second
+* `disk`            - Disk storage per broker in megabytes (Mi), gigabytes (Gi) or terabytes (Ti) (Default: 100000Mi)
+* `cpuUtilization`  - CPU utilization as a percentage (Default: 100)
+* `inboundNetwork`  - Inbound network throughput in kilobytes (KiB/s), megabytes (MiB/s) or gigabytes (GiB/s) per second (Default: 10000KiB/s)
+* `outboundNetwork` - Outbound network throughput in kilobytes (KiB/s), megabytes (MiB/s) or gigabytes (GiB/s) per second (Default: 10000KiB/s)
```

Comment:
Looks good. But the example below lists `KB/s` So should we change it there to `KiB/s`? Or add `KB/s` here?

Reply:
Above the bullet points I added a line about the "i" being optional. If that is not clear enough I can add both forms in the units brackets? WDYT?

Reply:
So this is my bad. I was misunderstanding the unit parsing code. We use K8s standard units, M for base 10 (5 G = 5000 MB) or Mi for base 2 (5 Gi = 5368.70 MB). I have updated the docs. Hope this is clearer (correct) now.